build.gradle.kts

```diff
@@ -59,6 +59,7 @@ dependencies {
   testImplementation("org.springframework.security:spring-security-test")
   testImplementation("org.testcontainers:postgresql:1.17.6")
   testImplementation("org.testcontainers:localstack:1.17.6")
+  testImplementation("com.amazonaws:aws-java-sdk-s3:1.12.13")
```

Comment:
odd this is, I don't have this on mine and everything works, Has this got something to do with the docker/local host stuff you did?